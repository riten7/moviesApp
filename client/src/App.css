@import '~antd/dist/antd.css';
body {
  font-size: 14px;
  font-family: 'Roboto', sans-serif;
  color: #888585;
  line-height: 24px;
}
body {
  font-size: 14px;
  background-color: #292727;
}
.App {
  text-align: center;
}
/* movie container */
.movieSearch {
  background-color: #3c3838;
  padding: 24px;
  min-height: 160px;
  display: flex;
  flex-flow: column nowrap;
  justify-content: center;
}
.search-form{
  display: flex;
  max-width: 65%;
  margin: 0 auto;
  width: 100%;
}
.search-form .ant-col:first-child {
  width: 100%;
}
.search-form .ant-col:last-child {
  width: 150px;
}
.search-form .ant-col:last-child .addMovieBtn {
  min-height: auto;
  margin: 0;
  background-color: #708a15;
  font-size: 32px;
  border-radius: 50%;
  height: 60px;
  width: 60px;
}
.search-form .ant-col:first-child input,
.search-form .ant-col:last-child button{
  border: none;
}
.search-form .ant-col:first-child input:focus,
.search-form .ant-col:last-child button:focus {
  outline: none;
  border: none;
  box-shadow: none;
}
.search-form .ant-col:first-child p{
  text-align: left;
  font-size: 16px;
  color: #888585;
  padding: 10px 0 0 20px;
  margin: 0;
  font-weight: 300;
}
.search-form .ant-col:first-child input:focus{
  background-color: #fff;
  color: #696565;
}
.movieSearch .searchInput {
  background-color: #696565;
  color: #f2e9e1;
  min-height: 60px;
  border-radius: 30px;
  padding: 0px 30px;
  font-size: 18px;
  font-weight: 300;
}

.movieSearch .addMovieBtn,
.movieDetail .editMovieBtn,
.movieDetail .deleteMovieBtn {
  background-color: #708b16;
  color: #f2e9e1;
  border-radius: 0.5em;
  min-height: 60px;
  margin: 0 12px;
}
.ant-btn {
  margin: 2px;
}
.gutter-row {
  padding: 12px;
}
.movieCard img{
  display: block;
  width: 100%;
}
.card-body {
  padding: 10px 16px;
}
.card-body p {
  color: #999;
}
.ant-card-body {
  padding: 10px;
  background-color: #708a14;
  text-align: center;
}
.ant-card-body .ant-card-meta-title,
.ant-card-body .ant-card-meta-description {
    color: #f2e9e1;
}
.movieList{
  display: flex;
  max-width: 82%;
  margin: 0 auto;
  padding: 50px 0px;
  /* justify-content: center; */
}
.movieList .ant-col {
  width: auto;
  max-width: initial;
}
.movieDetail {
  max-width: 85%;
  margin: 0 auto;
  padding: 50px 0;
}
.movieList .ant-card-bordered{
  border: 5px solid #000000;
  border-radius: 6px;
}
.movieList .ant-card-bordered:hover {
  box-shadow: 0px 0px 30px rgba(0,0,0,0.9);
  border-color: #fff;
}
.movieList .ant-card-body{
  padding: 20px 10px;
  background-color: #708a15;
  text-align: center;
}
.ant-card-meta-title{
  font-weight: 900;
  font-size: 14px;
}
.ant-card-meta-title ~.ant-card-meta-description{
  font-size: 12px;
}
.movieDetail .ant-row{
  display: flex;
  flex-flow: row wrap;
}
.movieDetail .ant-row .ant-col{
  padding: 30px;
}
.movieDetail .ant-row .ant-col:last-child{
  text-align: left;
}
.movieDetail .ant-row .ant-col:last-child h1{
  font-weight: 900;
  color: #708a15;
}
.movieDetail .ant-row .ant-col:last-child hr{
  border: 1px solid #3c3838;
}
.movieDetail .ant-row .ant-col:last-child .ant-btn{
  border: none;
  margin-top: 20px;
  padding: 0 30px;
}