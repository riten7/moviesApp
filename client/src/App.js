import React from 'react';
import { Switch, Route } from 'react-router-dom';
import MovieList from './containers/MovieList';
import Detail from './containers/detail';
import './App.css';

function App() {
  return (
    <div className="App">
      <Switch>
        <Route exact path='/' component={MovieList} />
        <Route exact path='/movie/:id' component={Detail} />
      </Switch>
    </div>
  );
}

export default App;
