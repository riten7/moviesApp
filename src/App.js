import React from 'react';
import { Switch, Route } from 'react-router-dom';
import MovieList from './containers/MovieList';
import MovieDetail from './containers/MovieDetail';
import './App.css';

function App() {
  return (
    <div className="App">
      <Switch>
        <Route exact path='/' component={MovieList} />
        <Route exact path='/movie/:id' component={MovieDetail} />
      </Switch>
    </div>
  );
}

export default App;
