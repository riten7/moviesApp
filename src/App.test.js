import React from "react";
import { Provider } from "react-redux";
import { BrowserRouter } from "react-router-dom";
import { render } from "enzyme";
import App from "./App";
import configureStore from "./store/reduxStore";

let store = configureStore();
beforeEach(() => {});
test("renders learn react link", () => {
  const wrapper = render(
    <Provider store={store}>
      <BrowserRouter>
        <App />
      </BrowserRouter>
    </Provider>
  );

  expect(wrapper.html()).toMatchSnapshot();
});