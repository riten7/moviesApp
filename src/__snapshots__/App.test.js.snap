// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `"<div class=\\"movie-container\\"><div class=\\"ant-row movieSearch\\"><div class=\\"search-form\\"><div class=\\"ant-col\\"><input type=\\"text\\" class=\\"ant-input searchInput\\" placeholder=\\"Search your movie here...\\" value><p>Search your favourite movies by typing.</p></div><div class=\\"ant-col\\"><button type=\\"button\\" class=\\"ant-btn addMovieBtn ant-btn-primary\\"><span role=\\"img\\" aria-label=\\"plus\\" class=\\"anticon anticon-plus\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" class=\\"\\" data-icon=\\"plus\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><defs><style/></defs><path d=\\"M482 152h60q8 0 8 8v704q0 8-8 8h-60q-8 0-8-8V160q0-8 8-8z\\"/><path d=\\"M176 474h672q8 0 8 8v60q0 8-8 8H176q-8 0-8-8v-60q0-8 8-8z\\"/></svg></span></button></div></div></div><div class=\\"ant-row movieList\\"><div class=\\"noMovieFound\\"><span role=\\"img\\" aria-label=\\"file-search\\" class=\\"anticon anticon-file-search\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" class=\\"\\" data-icon=\\"file-search\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M688 312v-48c0-4.4-3.6-8-8-8H296c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8h384c4.4 0 8-3.6 8-8zm-392 88c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8h184c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8H296zm144 452H208V148h560v344c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V108c0-17.7-14.3-32-32-32H168c-17.7 0-32 14.3-32 32v784c0 17.7 14.3 32 32 32h272c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8zm445.7 51.5l-93.3-93.3C814.7 780.7 828 743.9 828 704c0-97.2-78.8-176-176-176s-176 78.8-176 176 78.8 176 176 176c35.8 0 69-10.7 96.8-29l94.7 94.7c1.6 1.6 3.6 2.3 5.6 2.3s4.1-.8 5.6-2.3l31-31a7.9 7.9 0 000-11.2zM652 816c-61.9 0-112-50.1-112-112s50.1-112 112-112 112 50.1 112 112-50.1 112-112 112z\\"/></svg></span><p>No Movie Found</p></div></div></div>"`;
